bcda/cclf/metrics/metrics.go

```diff
@@ -0,0 +1,96 @@
+package metrics
```

Comment:
Interesting.  I wonder if we'd get the same benefit, with less code, if we used the New Relic transaction monitoring?  Is something like this useful? https://github.com/CMSgov/bcda-app/pull/111

Reply:
Unfortunately, the slick monitoring that we get for free from NewRelic really times RPC calls (HTTP, DB calls). To do in-process timing, like we're doing for CCLF ingestion, we need to orchestrate the transactions/segments.

We definitely should use this approach when timing the Postgres calls! If we can upgrade to 3.x, they have out of the box monitoring for Postgres, like we're using for HTTP tracing.

Reply:
Gotcha.  I'd love to see, as part of the `Acceptance Validation`, some results you've pulled out of New Relic after deploying this feature branch to `dev` and running the smoke test.

Reply:
Yup, will do!

Reply:
@mtrang1263 Sorry for being pedantic, when you said "orchestrate", did you mean "instrument"?